src/mpi/init/initthread.c

```diff
@@ -537,9 +537,6 @@ int MPIR_Init_thread(int *argc, char ***argv, int required, int *provided)
 #else
     MPIR_Process.attrs.tag_ub >>= 1;
 #endif
-    /* The bit for error checking is set in a macro in mpiimpl.h for
-     * performance reasons. */
-    MPIR_Process.tagged_coll_mask = MPIR_Process.attrs.tag_ub + 1;
```

Comment:
Changing to use a macro instead of this makes me a little nervous since different configurations have a different value for `tag_ub` (beyond the two that seem to be covered in the macro checks.